protocols/luci-proto-vxlan/htdocs/luci-static/resources/protocol/vxlan.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+'require form';
+'require network';
+'require tools.widgets as widgets';
+
+network.registerPatternVirtual(/^vxlan-.+$/);
+
+return network.registerProtocol('vxlan', {
+	getI18n: function() {
+		return _('VXLAN (RFC7348)');
+	},
+
+	getIfname: function() {
+		return this._ubus('l3_device') || 'vxlan-%s'.format(this.sid);
+	},
+
+	getOpkgPackage: function() {
+		return 'vxlan';
+	},
+
+	isFloating: function() {
+		return true;
+	},
+
+	isVirtual: function() {
+		return true;
+	},
+
+	getDevices: function() {
+		return null;
+	},
+
+	containsDevice: function(ifname) {
+		return (network.getIfnameOf(ifname) == this.getIfname());
+	},
+
+	renderFormOptions: function(s) {
+		var o;
+
+		o = s.taboption('general', form.Value, 'remote', _('Remote IPv4 address'), _('The IPv4 address or the fully-qualified domain name of the remote end.'));
+		o.optional = false;
+		o.datatype = 'or(hostname,ip4addr("nomask"))';
+
+		o = s.taboption('general', form.Value, 'local', _('Local IPv4 address'), _('The local IPv4 address over which the tunnel is created (optional).'));
+		o.optional = true;
+		o.datatype = 'ip4addr("nomask")';
+
+		o = s.taboption('general', form.Value, 'id', _('VXLAN network identifier'), _('ID used to uniquely identify the VXLAN'));
+		o.optional = true;
+		o.datatype = 'range(1, 16,777,216)';
```

Comment:
You need to remove the commas here, otherwise it is interpreted as `range(1, 16)`

Reply:
Fixed, thanks.